traveltimepy/dto/common.py

```diff
@@ -123,11 +123,23 @@ class Property(str, Enum):
     FARES = "fares"
 
 
-class SnapPenalty(str, Enum):
+class SnappingPenalty(str, Enum):
     ENABLED = "enabled"
     DISABLED = "disabled"
 
 
+class SnappingAcceptRoads(str, Enum):
+    BOTH_DRIVABLE_AND_WALKABLE = "both_drivable_and_walkable"
+    ANY_DRIVABLE = "any_drivable"
+
+
+class Snapping(BaseModel):
+    penalty: Optional[SnappingPenalty] = SnappingPenalty.ENABLED
+    accept_roads: Optional[SnappingAcceptRoads] = (
+        SnappingAcceptRoads.BOTH_DRIVABLE_AND_WALKABLE
+    )
```

Comment:
default values of `None` should work here too. This way it's easy to see the default value, but `None` wouldn't require changing if default value in api changes